yolo.py

```diff
@@ -174,10 +174,18 @@ def check_for_markdown(response):
     print(colored("The proposed command contains markdown, so I did not execute the response directly: \n", 'red')+response)
     sys.exit(-1)
 
+def missing_posix_display():
+  display = subprocess.check_output("echo $DISPLAY", shell=True)
+  return display == b'\n'
+
```

Comment:
missing posix display checks if anything is in the $DISPLAY env variable and returns true if it is empty.
I used it to remove the copy option in prompt_user_input() and also as a final check on the user input.

Reply:
Cool, I didn't know both macOS and Linux have `os.name="posix"`.